openedx/openedx-platform
# feat: management command to create topic links for the new provider from existing discussion blocks [BD-38]

## Description

Adds a management command that creates discussion topic links for the new openedx provider by copying over existing discussion topic ids. This allows using the existing topic structure in the new UI. 

This is for testing only. 

## Testing instructions

- For any course run this command to create discussion topic links for the openedx provider. 
- Switch to the openedx provider from the admin panel.
- Now you should see the discussions sidebar here: https://github.com/edx/frontend-app-learning/pull/762
